### PR TITLE
refactor: move to modules, core exports

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Move to modules and core exports and minor server/main style fix

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -42,37 +35,33 @@ end)
...
 RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     hasFitbit = false
-    PlayerData = {}
 end)
 
-RegisterNetEvent('QBCore:Player:SetPlayerData', function(val)
-    PlayerData = val
-    hasFitbit = fitbitCheck(PlayerData.items)
+RegisterNetEvent('QBCore:Player:SetPlayerData', function()
+    hasFitbit = fitbitCheck(QBX.PlayerData.items)
```
Comment (Manason) [inline client/main.lua]: Use the ox_inventory event to check for existence of an item

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (mafewtm) [reply client/main.lua]: Hopefully the new commit resolves this

File: client/main.lua [hunk 2]
```diff
@@ -35,19 +28,19 @@ end)
...
 RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
-    hasFitbit = fitbitCheck(QBX.PlayerData.items)
+    hasFitbit = exports.ox_inventory:Search('count', 'fitbit') >= 1
 end)
 
 RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     hasFitbit = false
 end)
 
 RegisterNetEvent('QBCore:Player:SetPlayerData', function()
```
Comment (Manason) [inline client/main.lua:51]: The event itself should be an ox_inventory change inventory invent

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 2 above)
Comment (mafewtm) [reply client/main.lua:51]: what event is this?

Comment (Manason) [reply client/main.lua:51]: https://overextended.dev/ox_inventory/Events/Client#ox_inventoryitemcount

Comment (mafewtm) [reply client/main.lua:51]: And how would this properly be incorporated across these three handlers with the item check

Comment (Manason) [reply client/main.lua:51]: If you want to know whether the player has a Fitbit or not, listen to inventory changes. The event tells you every time an item changes its count. So write an if condition to check if the item that changes count is 'Fitbit' and the count > 0

Comment (mafewtm) [reply client/main.lua:51]: Hows that?

**Pull request merged**